src/main/kotlin/racingcar/domain/TryNumber.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar.domain
+
+private const val MIN_TRY_NUMBER = 1
+private const val USE_COMPLETED_TRY_NUMBER = 0
+
+class TryNumber(value: Int) {
+    var value: Int = value
+        private set
+
+    init {
+        require(value >= MIN_TRY_NUMBER) { "시도횟수는 1이상인 수만 입력받을 수 있습니다. value = $value" }
+    }
+
+    fun consume() {
+        if (this.isComplete()) {
+            throw IllegalStateException("잔여 시도횟수가 없습니다.")
+        }
+
+        value--
+    }
+
+    fun isComplete(): Boolean = value == USE_COMPLETED_TRY_NUMBER
```

Comment:
이것도 한번 코틀린 감성을 얹어봅시다!
이 메서드는 해당 객체의 상태를 확인하는것으로 보이는데요.
코틀린의 프로퍼티는 상태를 나타내니까 요렇게 사용자 정의 getter를 이용하는 방법도 있습니다~!
```suggestion
    val isComplete: Boolean
        get() = value == USE_COMPLETED_TRY_NUMBER
```

Reply:
오... 코틀린스러움... 너무 좋네요. 😆 

지금 과정과는 조금 별개의 내용이지만, 지금처럼 `isComplete()` 함수를 `isComplete` 프로퍼티로 변경하게 되면, JPA를 사용한다는 가정에서 TryNumber Entity가 생성하는 테이블의 컬럼에 `isComplete 컬럼`이 생기는건지 아닌건지 궁금합니다. 🤔

관련된 내용을 이전에 들은 적이 있었던 것 같은데... 오래전에 봤던 내용이라 어디서 봤던 내용인지 기억이 잘 안나네요. 🥲 
(자바 진영의 경우 특정 애너테이션으로 특정 필드를 컬럼으로 생성하지 않는다고 명시해야했던 것 같은데... 코틀린은 별도로 애너테이션 붙일 필요가 없다고 했던 것 같은데.. 어렴풋이 기억이 나서 이 내용이 맞는지가 궁금하네요. 👀)

Reply:
엇 요거는 JPA에서 확인 한번 해봐야겠네요~! 저도 한번 확인해보고 코멘트 드릴게요~!

Reply:
jason이 발표하셨던 [어디 가서 코프링 매우 알은 체하기](https://youtu.be/ewBri47JWII?t=5582)에서 언급해주셨네요! 😃 
AccessType이 Field인 경우에은ㄴ `@Transient` 없이도 컬럼으로 인지하지 못하게 할 수 있다고 하네요! 😃

Reply:
아이코 이미 확인하셨었군요! ㅎㅎ